src/app/component/user/components/profile/edit-profile/edit-profile.component.scss

```diff
@@ -20,6 +20,73 @@ $main-font: OpenSans, sans-serif;
         color: #494A49;
     }
 
+    .privacy-wrapper {
+        border-bottom: 1px solid #E1E5E3;
+        margin-top: 40px;
+        padding-bottom: 40px;
+        
+        ul {
+            margin-bottom: 0;
+            padding: 0;
+        
+            li {
+                list-style: none;
+                margin-top: 28px;
+        
+                .label-wrapper {
+                    color: #494A49;
+                    cursor: pointer;
+                    font-family: OpenSans, sans-serif;
+                    font-size: 18px;
+                    line-height: 24px;
+                    margin: 0;
+                    position: relative;
+                    padding-left: 32px;
+        
+                    input {
+                        height: 0;
+                        opacity: 0;
+                        position: absolute;
+                        width: 0;
+                    }
+        
+                    input:checked ~ .checkmark {
+                        background-color: #13AA57;
+                    }
+        
+                    input:checked ~ .checkmark:after {
+                        display: block;
+                    }
+        
+                    .checkmark {
+                        border: 1px solid #13AA57;
+                        border-radius: 4px;
+                        height: 20px;
+                        left: 0;
+                        position: absolute;
```

Comment:
Why did you use the position `absolute`?

Reply:
It's actually not my code, but as i can see ti have to stylize a checkbox, so it hide input and replaced it with span